MTA_limit_trends/Comp/run_comp_grad_data_update.py

```diff
@@ -6,59 +6,57 @@
 #                                                                                           #
 #           author: t. isobe (tisobe@cfa.harvard.edu)                                       #
 #                                                                                           #
-#           last update: Feb 02, 2021                                                       #
+#           last update: May 06, 2024                                                       #
 #                                                                                           #
 #############################################################################################
 
 import os
 import sys
 import re
-import string
 import time
 import numpy
 import astropy.io.fits  as pyfits
 from astropy.io.fits import Column
 import Ska.engarchive.fetch as fetch
 import Chandra.Time
+import traceback
+import signal
+import argparse
 import getpass
 #
-#--- reading directory list
+#--- Define Directory Pathing
 #
-path = '/data/mta/Script/MTA_limit_trends/Scripts/house_keeping/dir_list'
-with open(path, 'r') as f:
-    data = [line.strip() for line in f.readlines()]
+BIN_DIR = "/data/mta/Script/MTA_limit_trends/Scripts"
+MTA_DIR = "/data/mta4/Script/Python3.11/MTA"
+DEPOSIT_DIR = "/data/mta/Script/MTA_limit_trends/Deposit"
+COMP_DIR = f"{DEPOSIT_DIR}/Comp_save"
+GRAD_DIR = f"{DEPOSIT_DIR}/Grad_save"
+DATA_DIR = "/data/mta/Script/MTA_limit_trends/Data"
 
-for ent in data:
-    atemp = re.split(':', ent)
-    var  = atemp[1].strip()
-    line = atemp[0].strip()
-    exec("%s = %s" %(var, line))
 #
 #--- append path to a private folder
 #
-sys.path.append(bin_dir)
-sys.path.append(mta_dir)
+sys.path.append(BIN_DIR)
+sys.path.append(MTA_DIR)
```

Comment:
The use of the MTA common functions has been removed in this PR. However the use of the fits_operations package is still necessary, so therefore the pathing to the /data/mta4/Script/Python3.11/MTA directory is still included.